src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php

```diff
@@ -275,7 +276,19 @@ public function bulkDeleteAction(Request $request)
      */
     protected function getErrorMessages()
     {
+        $iniConfig = $this->get('prestashop.core.configuration.ini_configuration');
+
         return [
+            UploadedImageConstraintException::class => [
+                UploadedImageConstraintException::EXCEEDED_SIZE => $this->trans(
+                    'Max file size allowed is "%s" bytes.', 'Admin.Notifications.Error', [
+                    $iniConfig->getUploadMaxSizeInBytes(),
+                ]),
```

Comment:
```suggestion
                    'Max file size allowed is "%s" bytes.', 'Admin.Notifications.Error',
                    [
                        $iniConfig->getUploadMaxSizeInBytes(),
                    ]
                ),
```

Reply:
What about 
```suggestion
                    'Max file size allowed is "%s" bytes.', 'Admin.Notifications.Error',
                    [$iniConfig->getUploadMaxSizeInBytes()]
                ),
```

This way we don't add 2 extra lines for brackets while there is a single element inside the array? 🤔  Wdyt ?

Reply:
IMO makes sense, thanks you both for suggestion!